tests/test_resource.py

```diff
@@ -234,7 +246,7 @@ def teardown_method(self):
         del environ["CORS_ORIGIN"]
 
     def make_cors_handler(self, origins: List[str] = None, req_origin: str = None) -> CORSResource:
-        an_event = defaultdict(MagicMock())
+        an_event: defaultdict = defaultdict(MagicMock())
```

Comment:
This type should be obvious as well - you don't have to annotate it that directly.